modules/core/src/effects/lighting/suncalc.js

```diff
@@ -0,0 +1,105 @@
+// sun position calculations are based on http://aa.quae.nl/en/reken/zonpositie.html formulas
+// and inspired by https://github.com/mourner/suncalc/blob/master/suncalc.js
+const PI = Math.PI;
```

Comment:
Is it necessary to define these constants? I personally find it easier to read if `Math.*` is directly used.